aimsprop/xyz.py

```diff
@@ -18,13 +18,13 @@ def parse_xyz(
     dt=20.0,
```

Comment:
should there be an option to input the widths into the function so that parsed xys also have atom widths?